apps/remix-ide/src/app/files/workspaceFileProvider.js

```diff
@@ -8,6 +9,7 @@ class WorkspaceFileProvider extends FileProvider {
     super('')
     this.workspacesPath = '.workspaces'
     this.workspace = null
+    this.event = new EventManager()
```

Comment:
instead of using `require('../../lib/events')` you could also use `const EventEmitter = require('events')` which is the oob solution. The previous one is no longer used for new features.

Reply:
ok

Reply:
it is not working with a change.

Reply:
you might need to update the function, it should work

Reply:
just check where it is used

Reply:
you can find example where it's being used there https://github.com/ethereum/remix-project/search?q=EventEmitter